### DISCUSSION TITLE
Configure new PTZ cameras

### DISCUSSION THREAD

**Discussion by DivanX10** [doitandbedone/ispyagentdvr-docker#488] [General]
There is no [Eufi Indoor Com 2K Pan & Tilt](https://ru.eufylife.com/products/kamery-dlya-doma/T8410322/T8410322/) camera in the PTZ list. The camera itself can be rotated to any position via the native Eufy app. Is it possible to add camera control to Agent DVR?

**Comment by doitandbedone** [comment]
@ispysoftware Any takes on this? I'm sure most implementations are generic. Not too sure if current support may be able to cover it even though the name is not explicit in app?

**Comment by doitandbedone** [comment]
@ispysoftware Any comments on this?

**Comment by ispysoftware** [comment]
you'd need to go through adding support for unlisted cameras on
https://www.ispyconnect.com/userguide-ptz.aspx
(for ispy but same applies to Agent)
